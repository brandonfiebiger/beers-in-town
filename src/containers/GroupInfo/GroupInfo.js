import React from 'react';
import ReactHtmlParser, { processNodes, convertNodeToElement, htmlparser2 } from 'react-html-parser';


export const GroupInfo = (props) => {

  return(
    <section>
      
    </section>
  )
}

export default GroupInfo;