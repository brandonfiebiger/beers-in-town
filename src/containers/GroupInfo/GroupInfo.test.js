import { GroupInfo } from './GroupInfo';
import { shallow, mount} from 'enzyme';
import React from 'react';

describe('GroupInfo', () => {

  let wrapper;
  let mockGroupProps

  beforeEach(() => {
    mockGroupProps = {
      name: 'joane'
    }
    wrapper = shallow(<GroupInfo groupProps={ mockGroupProps }/>)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should match the snapshot with a photo', () => {
    mockGroupProps = {
      name: 'joane',
      group_photo: {
        photo_link: 'yay'
      }
    }

    expect(wrapper).toMatchSnapshot()
  })
})