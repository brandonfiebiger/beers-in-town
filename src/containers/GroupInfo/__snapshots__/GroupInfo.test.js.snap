// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupInfo should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    groupProps={
        Object {
            "name": "joane",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          joane
</h2>,
        <h3>
          , 
</h3>,
        "",
        <article>
          
</article>,
        "",
      ],
      "className": "group-info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "joane",
        },
        "ref": null,
        "rendered": "joane",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            ", ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          undefined,
          ", ",
          undefined,
        ],
        "type": "h3",
      },
      "",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": "article",
      },
      "",
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            joane
</h2>,
          <h3>
            , 
</h3>,
          "",
          <article>
            
</article>,
          "",
        ],
        "className": "group-info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "joane",
          },
          "ref": null,
          "rendered": "joane",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              ", ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            undefined,
            ", ",
            undefined,
          ],
          "type": "h3",
        },
        "",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [],
          },
          "ref": null,
          "rendered": Array [],
          "type": "article",
        },
        "",
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;

exports[`GroupInfo should match the snapshot with a photo 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    groupProps={
        Object {
            "name": "joane",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          joane
</h2>,
        <h3>
          , 
</h3>,
        "",
        <article>
          
</article>,
        "",
      ],
      "className": "group-info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "joane",
        },
        "ref": null,
        "rendered": "joane",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            ", ",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          undefined,
          ", ",
          undefined,
        ],
        "type": "h3",
      },
      "",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
        },
        "ref": null,
        "rendered": Array [],
        "type": "article",
      },
      "",
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            joane
</h2>,
          <h3>
            , 
</h3>,
          "",
          <article>
            
</article>,
          "",
        ],
        "className": "group-info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "joane",
          },
          "ref": null,
          "rendered": "joane",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              ", ",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            undefined,
            ", ",
            undefined,
          ],
          "type": "h3",
        },
        "",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [],
          },
          "ref": null,
          "rendered": Array [],
          "type": "article",
        },
        "",
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
